models/news.models.js

```diff
@@ -29,3 +29,20 @@ exports.fetchArticles = () => {
       return rows;
     });
 };
+
+exports.fetchCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`,
+      [id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
No comments coming back from an article doesn't necessarily mean it doesn't exist, it might just have no comments 🤔 
If we could first check whether or not an article exists, then we would know whatever came back from this query is valid.

Reply:
I realised after the fact that I'd run into an issue here if the article had no comments. Will revisit!